Integrations/integration-MinervaLabsAntiEvasionPlatform.yml

```diff
@@ -0,0 +1,293 @@
+                                json=json_params)
+        if response.status_code is not 200:
+            return_error('Error while perfroming search for {}. More information: {}, {}'.format(search_url.rsplit('/')[1], response.status_code, response.reason))
+        demisto.results(response.json())
+
+
+    # ''' EXECUTION '''
+    session = get_session()
+    try:
+        def add_exclusion_command():
+            args = demisto.args()
+            exclusion_type = args.get('type')
+            exclusion_data = args.get('data')
+            exclusion_description = args.get('description')
+            groups_id = args.get('appliedGroupsId')
+            add_exclusion(exclusion_type, exclusion_data, exclusion_description, groups_id)
+
+
+        def add_vaccine_command():
+            args = demisto.args()
+            vaccine_name = args.get('name')
+            vaccine_desc = args.get('description')
+            monitor_only = args.get('isMonitorOnly')
+            add_vaccine(vaccine_name, vaccine_desc, monitor_only)
+
+
+        def search_process_command():
+            processes_search_url = BASE_URL + '/api/processes'
+            args = demisto.args()
+            search_param = args.get('param')
+            search_condition = args.get('condition')
+            search_value = args.get('value')
+            search(processes_search_url, search_param, search_condition, search_value)
+
+
+        def search_endpoint_command():
+            endpoint_search_url = BASE_URL + '/api/endpoints'
+            args = demisto.args()
+            search_param = args.get('param')
+            search_condition = args.get('condition')
+            search_value = args.get('value')
+            search(endpoint_search_url, search_param, search_condition, search_value)
+
+
+        LOG('command is {}'.format(demisto.command()))
+        if USERNAME and PASSWORD:
+            minerva_login = login()
+        else:
+            return_error('Failed to log in, no credentials were given')
+        if demisto.command() == 'fetch-incidents':
+            fetch_incidents()
+        elif demisto.command() == 'test-module':
+            if minerva_login.status_code == 200:
+                demisto.results('ok')
+            else:
+                return_error('Failed to log in. More information: {}, {}', minerva_login.status_code, minerva_login.reason)
+        elif demisto.command() == 'minerva-add-exclusion':
+            add_exclusion_command()
+        if demisto.command() == "minerva-add-vaccine":
+            add_vaccine_command()
+        if demisto.command() == 'minerva-search-process':
+            search_process_command()
+        if demisto.command() == 'minerva-search-endpoint':
+            search_endpoint_command()
+    except Exception, e:
+        LOG('Cannot perform the command: {}. Error: {}'.format(demisto.command(), e.message))
+        LOG.print_log()
+        return_error(e.message)
+    finally:
+        logout()
+  type: python
+  commands:
```

Comment:
Please add outputs - https://github.com/demisto/content/blob/master/docs/context_and_ouputs/README.MD

Reply:
@liorblob, I'm not sure context and outputs are relevant for this kind of integration